server.js

```diff
@@ -21,9 +15,117 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+    {"Welcome to the ultimate BOOK REVIEW LIST!":  [
+{
+      "books": "All books available", 
+      "/books/id/:id": "Search for a book by specific id",
+      "/books/title/:title": "Search for a book by specific title",
+      "/books/authors/:authors": "Search for book/books written by specific author (alone or with others)",
+      "/books/language_code/:language_code": "Search for books available in specific language",
+    }
+    ]
+  } );
 });
 
+//All books available
+app.get("/books", (req, res) => {
+  res.status(200).json({
+    data: books,
+    success: true,
+  });
+  });
+
+
+  //Endpoint 1: Searchning for a specific book by bookid
+app.get('/books/id/:id', (req, res) => {
+  const { id } = req.params
+  const bookById = books.find ((item) => item.bookID === +id)
+
+  if (!bookById) {
+    res.status(400).json({
+      ressponse: "There is no book with that ID number", 
+      success: false,
+    })
+  } else {
+    res.status(200).json({
+      response: bookById, 
+      success: true,
+    })
+  }
+})
+
+  //Endpoint 2: Searching for a specific  book title
+  app.get("/books/title/:title", (req, res) => {
+    const { title } =req.params;
+    
+    const bookByTitle = books.find((book ) => book.title.toLowerCase()=== title.toLowerCase());
+    
+    if (!bookByTitle){
+      res.status(200).json({
+        data: "Succesful search, but that book title is misspelled or not available here.",
+        success: false
+      });
+    } else {
+      res.status(200).json({
+        data: bookByTitle, 
+        success: true,
```

Comment:
I'm not sure you should have the same status code on line 65 and line 70? I'm thinking the more "correct" way is to throw some sort of 400-status (404) on line 65?